web/src/beta/lib/core/engines/Cesium/Feature/Model/index.tsx

```diff
@@ -107,6 +109,7 @@ export default function Model({ id, isVisible, property, geometry, layer, featur
         uri={actualUrl}
         scale={scale}
         shadows={shadowMode(shadows)}
+        customShader={!pbr ? NonPBRLightingShader : undefined}
```

Comment:
Do we need `pbr === false ? ...` like you set for 3d tiles? `!pbr` will apply NonPBRLightingShader if pbr is undefined.

Reply:
Fixed:pray: [fix](https://github.com/reearth/reearth/pull/517/commits/c88d84f7528684a7c661c1e6463b29d8c853db32)